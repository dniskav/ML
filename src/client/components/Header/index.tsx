import React from 'react';
import styled from 'styled-components';
import Logo from '../Logo/';
import SearchBox from '../SearchBox/';

const StyledHeader = styled.div`
    align-items: center;
    border: none;
    display: flex;
    height: 50px;
    justify-content: center;
    width: 1130px;
    background: #ffe600;
`;

const Header = () => {

  return(
    <StyledHeader>
      <Logo />
      <SearchBox />
    </StyledHeader>
  )
};

export default Header;
