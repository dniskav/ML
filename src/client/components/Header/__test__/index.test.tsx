import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Header from '../index';

jest.mock('react-redux');

describe('<Header />', () => {
  it("render success", () => {
    const wrapper: ShallowWrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
