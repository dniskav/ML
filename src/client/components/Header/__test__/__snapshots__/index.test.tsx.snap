// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> render success 1`] = `ShallowWrapper {}`;
